### DISCUSSION TITLE
Trust and Transparency WG

### DISCUSSION THREAD

**Discussion by raghavrmadya** [filecoin-project/notary-governance#605] [T&T WG]
As the Trust and Transparency lead for Fil+, I am initiating a working group to engage the community on issues related to DataCap abuse. Any relevant issues including trends, flags, and approaches to mitigate DataCap abuse will be discussed in this WG, and updates from this group will be provided at governance calls. 

The engagement is proposed to begin with trust and transparency-focused slack channel. Weekly or Bi-weekly calls will be discussed as an option if there is sufficient interest and topics to be discussed. 

Please add to the discussion any ideas or topics you have related to this WG.

**Comment by raghavrmadya** [comment]
The slack channel has been created on Filecoin slack for the community to join - #fil-plus-trust-wg

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
now that the T&T is going to make active governance decisions 

https://github.com/filecoin-project/notary-governance/issues/833

we should move this forward and "legalize" the working group by properly incorporating it into the governance process. Define its roles and responsibilities in the governance process etc.

**Comment by raghavrmadya** [comment]
The T&T WG does not make governance decisions. The WG merely creates a space to flag cases and discussions related to DC abuse. Anyone can create an issue; the conclusions are not by me but the broader community. That said, I do agree formalizing some of the work we do, as defined by actually doing it since the WG was initiated, will be helpful. That is the intention with the T&T DAO
